types/jsreport-core/index.d.ts

```diff
@@ -65,50 +84,192 @@ declare namespace JsReport {
         directory: string;
     }
 
-    interface Reporter {
-        use(extension: Extension | ExtensionDefinition): Reporter;
-    }
-
-    interface Reporter {
-        afterRenderListeners: ListenerCollection;
-        afterTemplatingEnginesExecutedListeners: ListenerCollection;
-        beforeRenderListeners: ListenerCollection;
-        documentStore: DocumentStore;
-        initializeListeners: ListenerCollection;
+    interface ReporterInstance {
+        defaults?: any;
```

Comment:
`any` is not a good type here. and I also could not find any thing about this at <https://github.com/jsreport/jsreport-core/blob/master/lib/optionsSchema.js>.

Reply:
https://github.com/jsreport/jsreport-core/blob/b3f907cc3b8a4fe18dc91f48430b2df6e691264c/lib/reporter.js#L66

Reply:
Here. <https://github.com/jsreport/jsreport-core/blob/b3f907cc3b8a4fe18dc91f48430b2df6e691264c/lib/reporter.js#L40-L45>

Reply:
I don't this is resolved yet. `any` is not a good type.